src/CLI/actioner/sonic-cli-show-techsupport.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+import sys
+import time
+import json
+import ast
+import re
+from scripts.render_cli import show_cli_output
+from rpipe_utils import pipestr
+import cli_client as cc
+import urllib3
+
+api = cc.ApiClient()
+urllib3.disable_warnings()
+plugins = dict()
+
+
+def register(func):
+    """Register sdk client method as a plug-in"""
+    plugins[func.__name__] = func
+    return func
+
+def call_method(name, args):
+    method = plugins[name]
+    return method(args)
+
+def invoke(func, args):
+    body = None
+    aa = cc.ApiClient()
+
+    # Gather tech support information into a compressed file
+    if func == 'rpc_sonic_show_techsupport_sonic_show_techsupport_info':
+        keypath = cc.Path('/restconf/operations/sonic-show-techsupport:sonic-show-techsupport-info')
+        if args is None:
+            body = {"sonic-show-techsupport:input":{"date": None}}
+        else:
+            body = {"sonic-show-techsupport:input":{"date": args[0]}}
+        return aa.post(keypath, body)
+    else:
+        print("%Error: not implemented")
+        exit(1)
+
+
+def run(func, args):
+
+    try:
+        api_response = invoke(func, args)
+
+        if api_response.ok():
+            response = api_response.content
+
+            if response is None:
+                print ("ERROR: No output file generated: \n"
+                       "Invalid input: Incorrect DateTime format")
+
+	    else:
+		if func == 'rpc_sonic_show_techsupport_sonic_show_techsupport_info':
+                    if not response['sonic-show-techsupport:output']:
+                        print("ERROR: No top level output object: no output file available")
```

Comment:
Let's not provide internal details like top level object, second level object etc in the ERROR message.

Reply:
I have removed the "X level object" text from the error case print statements.